virtcontainers/clh.go

```diff
@@ -472,9 +474,43 @@ func (clh *cloudHypervisor) hotplugAddDevice(devInfo interface{}, devType device
 
 }
 
+func (clh *cloudHypervisor) hotplugRemoveBlockDevice(drive *config.BlockDrive) error {
+	cl := clh.client()
+	ctx, cancel := context.WithTimeout(context.Background(), clhHotPlugAPITimeout*time.Second)
+	defer cancel()
+
+	_, _, err := cl.VmmPingGet(ctx)
+	if err != nil {
+		return openAPIClientError(err)
+	}
+
+	driveID := clhDriveIndexToID(drive.Index)
+
+	if drive.Pmem {
+		err = fmt.Errorf("pmem device hotplug remove not supported")
```

Comment:
Have you actually tried it and it does not work? Or do you mean it's not yet supported from Kata?
I'm asking because that should work out of the box, as long as you use `remove-device` with the id corresponding to the `pmem` device.

Reply:
I means it is not yet supported in kata. I believe you are right, it should work out of the box with `remove-device` for both `pmem` and `disk`. It is mostly a placeholder before I add the support for `pmem` hotplug/unplug in Kata.